src/pages/layer-2.js

```diff
@@ -406,19 +406,34 @@ const Layer2Page = ({ data }) => {
         </StatsContainer>
       </PaddedContent>
 
+      <PaddedContent id="what-is-layer-2">
```

Comment:
Noting this layout currently looks poor - the focus was on content not design 😅 

This top section probably shouldn't be full width:
![Image 2022-03-24 at 10 42 22 PM](https://user-images.githubusercontent.com/8097623/160061541-e30aaf58-393f-40d9-a74b-182a36808437.jpg)

Reply:
Today we will look into it.